DataReducing.php

```diff
@@ -0,0 +1,103 @@
+<?php
+declare(strict_types=1);
+
+/**
+ * Input array must have two levels
+ *      - level 1 key types: int
+ *      - level 2 key types: int
+ *
+ * @param array $arr
+ * @param string $userInput
+ * @return array
+ */
+function removeDuplicates(array $arr, string $userInput): array
+{
+    $columnKey = $arr[0][0]; //holds the working column corresponding key, used for parsing the arrays from level 2
```

Comment:
the default value here is so confusing. just set it to empty string.

Reply:
solved